### ISSUE TITLE
ValueError: invalid literal for int() with base 10: 'auto'

### ISSUE THREAD

**Issue by trevorphillipscoding** [hhatto/autopep8#515]
---

## Python Code
```python

Traceback (most recent call last):
  File "/Users/trevorphillips/.local/share/virtualenvs/taxbot-web-l9QEaBVw/bin/autopep8", line 8, in <module>
    sys.exit(main())
  File "/Users/trevorphillips/.local/share/virtualenvs/taxbot-web-l9QEaBVw/lib/python3.7/site-packages/autopep8.py", line 4176, in main
    args = parse_args(argv[1:], apply_config=apply_config)
  File "/Users/trevorphillips/.local/share/virtualenvs/taxbot-web-l9QEaBVw/lib/python3.7/site-packages/autopep8.py", line 3650, in parse_args
    parser = read_config(args, parser)
  File "/Users/trevorphillips/.local/share/virtualenvs/taxbot-web-l9QEaBVw/lib/python3.7/site-packages/autopep8.py", line 3761, in read_config
    value = config.getint(section, k)
  File "/usr/local/Cellar/python/3.7.6_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/configparser.py", line 818, in getint
    fallback=fallback, **kwargs)
  File "/usr/local/Cellar/python/3.7.6_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/configparser.py", line 808, in _get_conv
    **kwargs)
  File "/usr/local/Cellar/python/3.7.6_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/configparser.py", line 802, in _get
    return conv(self.get(section, option, **kwargs))
ValueError: invalid literal for int() with base 10: 'auto'
```

Command Line
```shell
$ autopep8 -i -r .
```

## Your Environment
* Python version: Python 3.7.6
* autopep8 version: autopep8 1.4.4 (pycodestyle: 2.5.0)
* Platform: macOSX

**Comment by MingleiLI** [comment]
I have the same problem. 

Python version: Python 3.7.6
autopep8 version: autopep8 1.4.4 (pycodestyle: 2.5.0)
Platform: Ubuntu

**Comment by hhatto** [comment]
Thanks for reporting.

Could you tell me what the configuration file looks like?
and, Could you share the execution log of `autopep8` with `-vvv` option.

@trevorphillips @MingleiLI

**Comment by trevorphillips** [comment]
@hhatto I did a reinstall of my modules. (I deleted my pipenv files and just did a reinstall from my requirements.txt) and everything works as expected now.

I do have a configuration file that looks like this.
`[pep8]
exclude = .git,__pycache__,*/migrations/*,node_modules/*`

**Comment by imdeepmind** [comment]
Any update on this issue? I'm facing this same issue.

**Comment by hhatto** [comment]
@imdeepmind 
What is the version of autopep8? 1.4.4? 1.5.4 latest?

And, Can you indicate which repositories are experiencing specific problems?

**Comment by hhatto** [comment]
No reply, so I'll close it once.
If the problem still occurs, I will reopen it.

**Comment by hhatto** [comment]
@trevorphillipscoding @MingleiLI @imdeepmind 
This problem has been fixed in version 1.7.1.
Please try the latest version.
